docs/bill_of_materials_orin.md

```diff
@@ -0,0 +1,74 @@
+# Bill of Materials - Jetson Orin Nano version
+
+This page lists all the parts you need to build the open-source JetBot with the new [**Jetson Orin Nano 8GB Developer Kit**](https://developer.nvidia.com/embedded/learn/get-started-jetson-orin-nano-devkit), along with purchasing links from popular vendors.
```

Comment:
Maybe add a note here about when to use Jetson Orin Nano, and about Jetson Nano EOL

Reply:
Good idea, will do.